cip/CIP2015-06-24-call-procedures.adoc

```diff
@@ -0,0 +1,250 @@
+= CIP2015-06-24 Calling Stored Procedures
+:numbered:
+:toc:
+:toclevels: 3
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Stefan Plantikow <stefan.plantikow@neotechnology.com>
+*Author:* Nigel Small <nigel.small@neotechnology.com>
+*Author:* Petra Selmer <petra.selmer@neotechnology.com>
+*Author:* Rickard Öberg <rickard.öberg@neotechnology.com>
+
+[abstract]
+.Abstract
+
+This proposal introduces stored procedures as a new concept to Cypher and proposes a new facility
+for calling them.
+
+User-defined functions are a separate concern that might be addressed in a future CIP.
+
+toc::[]
+
+== Motivation & Background
+
+While Cypher is a very expressive graph query language, its declarative nature and scope is currently not sufficient for
+expressing certain classes of graph traversals, like traversals relying on using specialized data structures or on accessing third-party functionality.
```

Comment:
'on accessing third-party' sounds clumsy and reads badly. I think removal of 'on' will fix it